README.md

```diff
@@ -51,6 +51,7 @@ By default this plugin operates over [Sublime Text](http://www.sublimetext.com)
 
 | name | type | description |
 |---|---|---|
+| `config_location` | `string` | Folder path to save SyncSetting specific info. E.g.: Logs, Last Upload Time, etc
```

Comment:
Please, update documentation indicating, this property must be an absolute path.